lib/node/node.ts

```diff
@@ -83,3 +85,28 @@ if (httpClient && httpClient.ClientRequest) {
     }
   };
 }
+
+function patchNextTick() {
+  let setNative = null;
+
+  function scheduleTask(task: Task) {
+    const args = task.data;
+    args[0] = function() {
+      task.invoke.apply(this, arguments);
+    };
+    setNative.apply(process, args);
```

Comment:
Shouldn't this be `setNative.apply(self, args)` using the `self` variable coming from `patchMethod`?  Not actually a problem since `nextTick` appears to bind callbacks to `global`, but still